components/03-modules/dashboard-nav/_dashboard-nav.scss

```diff
@@ -0,0 +1,50 @@
+$dashboard-nav-padding: 15px !default;
+$dashboard-nav-margin : 15px !default;
+
+.dashboard-nav {
+    display: flex;
+    flex-direction: column;
+    background-color: $gray-lighter;
+    padding: $dashboard-nav-padding;
+    margin-bottom: $dashboard-nav-margin;
+
+    &__content {
+        padding: $dashboard-nav-padding 0;
+        border-top: 1px solid $gray;
+    }
+
+    &__item {
+        padding: 15px;
+        transition: $transition-base;
+
+        &:hover {
+            background-color: $white;
+        }
+
+        &:first-child {
+            padding-top: 15px;
+        }
+
+        &:last-child {
+            padding-bottom: 15px;
+        }
+    }
+
+    &__title {
+        margin-bottom: $dashboard-nav-margin;
+        padding-bottom: 10px;
+        border-bottom: 1px solid $gray;
+    }
+
+    .current {
+        border-left: 2px solid $red;
+        color: $red;
+        padding-left: 23px;
+        position: relative;
+        margin-left: -15px;
+
+        &:hover {
+            background-color: inherit;
+        }
+    }
+}
```

Comment:
Inside this nav items you'll be have the link tag (see magento dashboard), please add it and style

Reply:
class added, now it's `dashboard-nav__link`

Reply:
Ok. Could you please add `<a>` tags inside the <li> elements to see it works :)

Reply:
Sure I can but must create new `list` variant

Reply:
done, added

Reply:
👍